src/OrchardCore.Modules/OrchardCore.Workflows/Views/WorkflowType/Index.cshtml

```diff
@@ -64,14 +87,70 @@
                     </div>
                 </li>
             }
-        </ul>
-    }
+        }
+        else
+        {
+            <li>
+                <div class="alert alert-info" role="alert">
+                    @T["<strong>Nothing here!</strong> Your search returned no results."]
+                </div>
+            </li>
+        }
+    </ul>
+</form>
+@await DisplayAsync(Model.Pager)
+
+<script at="Foot" type="text/javascript">
+    $(function () {
+        $('select').selectpicker();
+
+        var actions = $("#actions");
+        var items = $("#items");
+        var filters = $(".filter");
+        var selectAllCtrl = $("#select-all");
+        var selectedItems = $("#selected-items");
+        var itemsCheckboxes = $(":checkbox[name='itemIds']");
+
+        $(".filter-options input").on("change", function () {
+            $("[name='submit.Filter']").click();
+        });
+
+        $(".dropdown-menu .dropdown-item").on("click", function () {
+            if ($(":checkbox[name='itemIds']:checked").length > 1) {
+                var $this = $(this);
+                confirmDialog({title: $this.data('title'), message: $this.data('message'), callback: function(r) {
```

Comment:
@jpiquot added a new syntax. Can we still use this one or do we have to use the new one everywhere?

Reply:
You mean @jptissot :)

Spread syntax